po/es.po

```diff
@@ -837,10 +840,12 @@ msgstr "Acercar"
 #: src/MainWindow.vala:150
 msgid "<b>Note:</b> Disabling JavaScript will likely break many sites"
 msgstr ""
+"<b>Nota:</b> Deshabilitando JavaScript probablemente dejen de funcionar "
```

Comment:
Change this to `<b>Nota:</b> si desactiva`…
This is an improper use of _gerundio_ in Spanish. Also note that there is no capital letter after `:`